research/src/components/checkbox-anatomy.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import './anatomy.css'
+
+export default () => {
+  return (
+    <div className="component-anatomy">
+      <host>
+        <part name="label">
```

Comment:
Do the frameworks really have a label built into the checkbox control? From the webplat perspective the label is its own label and the relationship is made either through descendant or for attribute. I'm not saying this isn't the case but I think this may warrant a discussion/issue.

Reply:
Typically the label is *optional* or there is a mechanism to make the label visible only to AT, but generally Checkbox components have a simple, first-class mechanism to create a label for the checkbox.

I anticipated discussion here - it certainly differs from the way the platform works but it *is* aligned with the research of component libraries.

Reply:
And it is tied to the individual host of type `checkbox`? I think I can live with this although I find it kind of awkward but I get your overall point and we always recommend that folks have a label.